Usbipd/WslDistributions.cs

```diff
@@ -133,6 +133,15 @@ public static string GetVMSwitchName()
                     var isRunning = match.Groups[3].Value == "Running";
                     var version = uint.Parse(match.Groups[4].Value, CultureInfo.InvariantCulture);
 
+                    var networkingModeResult = await ProcessUtils.RunCapturedProcessAsync(WslPath, new [] { "--distribution", name, "--", "wslinfo", "--networking-mode", "-n" }, Encoding.UTF8, cancellationToken);
+                    var hostnameResult = await ProcessUtils.RunCapturedProcessAsync(WslPath, new [] { "--distribution", name, "--", "hostname", "-I" }, Encoding.UTF8, cancellationToken);
```

Comment:
This could possibly return more than one IP address? If so, this needs tweaking to handle such a scenario